pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -434,6 +434,7 @@ func (r *ReconcileCnsFileAccessConfig) configureVolumeACLs(ctx context.Context,
 	vSanFileShareNetPermissions = append(vSanFileShareNetPermissions, vsanfstypes.VsanFileShareNetPermission{
 		Ips:         tkgVMIP,
 		Permissions: vsanFileShareAccessType,
+		AllowRoot:   true,
```

Comment:
Setting this in the spec should not have any impact while deleting the volume ACLs, correct? As this method is common for both creation and deletion.

Reply:
Yes, setting here has no impact, since the API honors the delete flag in `CnsNFSAccessControlSpec`